src/note.ts

```diff
@@ -527,6 +527,11 @@ export abstract class Note extends Tickable {
    * @returns this
    */
   addModifier(modifier: Modifier, index: number = 0): this {
+    // Backwards compatibility with 3.0.9.
+    if (typeof index === 'string') {
+      index = parseInt(index);
```

Comment:
Consider logging a deprecated message. Parameter should be `index: number | string = 0`

Reply:
Well we shouldn't accept strings here. The API should officially be index: number = 0.

However, legacy code might actually pass in a string. That's what VexTab did. So I'm just going to parse it here. Sure I can log a warning.